docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,86 @@
+.. post:: Jan 20, 2018
+   :tags: community updates, core team
+   :author: Mikey Ariel
+
+Write the Docs 2018 Kick-Off
+============================
+
+Greetings, global documentarians!
+
+Mikey here, writing from the deep, dark cold European winter, but looking forward to spring and summer, full of sunshine and docs!
+Following Eric's winter-warming `2017 Stats <http://www.writethedocs.org/blog/write-the-docs-2017-stats/>`_ post, I'd like to highlight some of the activities, changes, and experiments that you can expect in 2018.
+
+These efforts are a part of optimizing our work in supporting and growing the community, and the also aim to keep things fresh and exciting for our documentarians.
+
+We hope you'll enjoy the new things we're trying. If you'd like to contribute, please write to us at support@writethedocs.org and let us know!
+
+So, new year - new...
+
+Core team
+---------
+
+Since the original Write the Docs conference was held in 2012, the folks behind the scenes have contributed their time, efforts, and talent on a volunteer basis, with tremendous results. Over the years, the community grew to a point where we needed to develop a system that will support the community in the long run, without burning out the people who are filling key global roles.
+
+Last year we decided to try setting up a formalized a small, part-time paid staff with the goal of consolidating various roles and responsibilities that could be handled globally, such as web administration, swag coordination, staff wrangling, and meetup overseeing.
+
+So far the experiment has been successful! Here are the core team members and their respective responsibilities:
+
+- Eric Holscher: Co-founder, North America conferences, sponsorship, finance
+- Mikey Ariel: Europe conferences, brand and design, swag, communication
+- Samuel Wright: Support, website/GitHub, meetups, CFP and speakers (Europe)
+- Kelly O'Brien: Newsletter, job board, CFP and speakers (North America)
+
+These focus areas are on an including-but-not-limited-to basis, since there are lots of smaller, ad-hoc tasks that the core team runs into throughout the year. You can find a high-level overview of the teams and roles in :doc:`Team </team>`, and if you want to learn more about the organization behind the community, you can read :doc:`About Us </about-us>`.
+
+Job opportunities
+-----------------
+
+In recent years we noticed that an increasing number of documentarians in our communities are looking to evolve their careers, and that many employers are seeking documentation-friendly folks to add to their organizations.
```

Comment:
"looking to evolve their careers" makes me go ewww (sorry). How about the less poetic "looking for work" :-D